linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -78,18 +75,18 @@ impl SharedLocalKubernetesNetTestingConfig {
                 .join("release");
             if binaries_dir.exists() {
                 // If release exists, use those binaries
-                Self {
-                    network,
-                    binaries: Some(Some(binaries_dir)),
-                }
-            } else {
-                // If release doesn't exist, pass None to build binaries
-                // from within Docker container
-                Self { network, binaries }
+                binaries = Some(Some(binaries_dir));
```

Comment:
nit: While we're cleaning up, can we (finally) create an enum for this mysterious double option type?

Reply:
Done: https://github.com/linera-io/linera-protocol/pull/1622/commits/44ec99382b820c97f746022f147e6fe05bda8138
I also removed the option entirely from `SharedLocalKubernetesNetTestingConfig::new` since it was `None` everywhere.